src/webui/components/Header/index.js

```diff
@@ -158,7 +159,7 @@ export default class Header extends React.Component {
             <img src={ this.state.logo } className={ classes.logo } />
           </Link>
           <figure>
-            npm set registry { registryURL }
+            npm set { this.state.scope }registry { registryURL }
```

Comment:
One thing, the scope is the text you provide in YAML config, i.e. **@sergiohgz**, but I don't find where you add a colon between provided scope and `registry` string, so it will appear as `npm set @sergiohgzregistry localhost`

Reply:
This gets done in src/webui/components/Header/index.js when initialising the state object:
``` JavaScript
scope: (window.VERDACCIO_SCOPE) ? `${window.VERDACCIO_SCOPE}:` : ''
```

Reply:
This should be at render and not be saved in state, so you can reuse that variable

Reply:
Yes. Please move it to render method.